### ISSUE TITLE
Hide tab navigation bar in Data Editor

### ISSUE THREAD

**Issue by DannyDeLoach** [dbeaver/dbeaver#35729]
It would be helpful to have additional customization options that allow users to further simplify the interface as desired.

In the Data Editor, I would like the option to hide the Tab Navigator Bar. I'm not sure that is the official name. This picture shows the bar I'm referring to.

![2024-09-26 17-21-57](https://github.com/user-attachments/assets/bd8f35cc-f1a0-4c99-9953-c608dbab18a4)

I have searched for a setting in Preferences that allows me to hide this toolbar, but I can't find any.

**Comment by E1izabeth** [comment]
Hi, @DannyDeLoach. There is no such preference yet. Could you tell me why do you want to hide it?

**Comment by DannyDeLoach** [comment]
Hi @E1izabeth. Thanks for responding. I prefer a UI that is as simple and uncluttered as possible. This allows me to better focus on the data. For powerful, complicated software like DBeaver, I recognize there are significant limits to how much the UI can be simplified and still have access to essential functionality. I'm asking for the ability to hide this UI element for two reasons: I don't use it, and its relatively large size means removing it would have an equally large impact on simplifying the appearance of the UI.

**Comment by E1izabeth** [comment]
Thanks for your request.
Connected with #35267

**Comment by varun-dhawan** [comment]
Agree, there are so many options in dbeaver console, while being useful , these are also distracting. Allowing to hide some of the elements such as 'Database Navigator' is make the life for users simple. Thanks for prioritizing this.

Version: 22.1.4

**Comment by E1izabeth** [comment]
We've moved the breadcrumbs to the bottom status bar. As I understand, it revolves the issue.
